README.rst

```diff
@@ -1,104 +1,138 @@
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
+- **Microsites:** Store the microsite configuration.
+- **Routes:** Configure the URL for a tenant.
+- **Tenant configs:** Store the configuration for each tenant.
+- **Tenant organizations:** Link each organization with one or multiple tenants.
 
-.. code-block:: yaml
+Add ``EDNX_USE_SIGNAL = True`` in each microsite/tenant that wants to use the plugin. 
+ 
+Compatibility Notes
+--------------------
 
-  GET_BRANDING_API: 'eox_tenant.edxapp_wrapper.backends.branding_api_h_v1'
-  GET_CERTIFICATES_MODULE: 'eox_tenant.edxapp_wrapper.backends.certificates_module_i_v1'
-  GET_SITE_CONFIGURATION_MODULE: 'eox_tenant.edxapp_wrapper.backends.site_configuration_module_i_v1'
-  GET_THEMING_HELPERS: 'eox_tenant.edxapp_wrapper.backends.theming_helpers_h_v1'
-  EOX_TENANT_EDX_AUTH_BACKEND: "eox_tenant.edxapp_wrapper.backends.edx_auth_i_v1"
-  EOX_TENANT_USERS_BACKEND: 'eox_tenant.edxapp_wrapper.backends.users_i_v1'
-  EDXMAKO_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.edxmako_h_v1'
-  UTILS_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.util_h_v1'
++------------------+----------------+
+| Open edX Release | Version        |
++==================+================+
+| Ironwood         | < 3.0          |
++------------------+----------------+
+| Juniper          | >= 3.0 < 4.0   |
++------------------+----------------+
+| Koa              | >= 4.0 <= 5.1.3|
++------------------+----------------+
+| Lilac            | >= 4.0 < 6.2   |
++------------------+----------------+
+| Maple            | >= 6.0 < 11.6  |
++------------------+----------------+
+| Nutmeg           | >= 6.2         |
++------------------+----------------+
+| Olive            | >= 8.0         |
++------------------+----------------+
+| Palm             | >= v10.0.0     |
++------------------+----------------+
+| Quince           | >= v11.0.0     |
++------------------+----------------+
+
+**NOTE**: Since the 6.2 version, eox-tenant does not support Django 2.2
+
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
+
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-tenant version.
 
-**Juniper**
+**Maple**
 
-For version >= 3.4
+For version  11.X compatible
 
 .. code-block:: yaml
 
-  GET_OAUTH_DISPATCH_BACKEND: 'eox_tenant.edxapp_wrapper.backends.oauth_dispatch_j_v1'
+  EOX_TENANT_EDX_AUTH_BACKEND = "eox_tenant.edxapp_wrapper.backends.edx_auth_i_v1"
+
+Those settings can be changed in ``eox_tenant/settings/common.py`` or, for example, in the instance settings.
 
-**Koa (optional)**
+**NOTE**
 
+For version > 10.0.0 you need to enable eox-tenant adding in the LMS configuration:
```

Comment:
```suggestion
ℹ️ For version > 10.0.0 you need to enable eox-tenant adding in the LMS configuration:
```

Reply:
The icon is not supported by .rst so I'm using 🚨 instead.